var/spack/repos/builtin/packages/astyle/package.py

```diff
@@ -32,9 +32,7 @@ class Astyle(MakefilePackage):
     """
 
     homepage = "http://astyle.sourceforge.net/"
-    # homepage is dead use Gentoo Mirror
-    # url = "http://downloads.sourceforge.net/project/astyle/astyle/astyle%202.04/astyle_2.04_linux.tar.gz"
-    url = "http://distfiles.gentoo.org/distfiles/astyle_3.0.1_linux.tar.gz"
+    url = "https://sourceforge.net/projects/astyle/files/astyle/astyle%203.0.1/astyle_3.0.1_linux.tar.gz"
```

Comment:
not sure if there is any difference between this link and the one which was commented. I guess i would simply uncomment one line and comment another.

Reply:
Just fine for me!

Reply:
Changed it accordingly! Is working like a charm for me :D

Reply:
> Changed it accordingly!

you actually did not but it's fine 😄